sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -656,13 +656,15 @@ protected synchronized void applyEnabledState(boolean enabled) {
 
         /* If we enabled the service. */
         if (enabled) {
+            mChannel.addGroup(criticalAnalyticsGroupName("Critical"), getTriggerCount(), getTriggerInterval(), getTriggerMaxParallelRequests(), null, getChannelListener());
```

Comment:
```suggestion
            mChannel.addGroup(ANALYTICS_CRITICAL_GROUP, getTriggerCount(), getTriggerInterval(), getTriggerMaxParallelRequests(), null, getChannelListener());
```

Replace `getTriggerInterval` to new const